src/libServer/IsolatedServer.cpp

```diff
@@ -443,6 +443,52 @@ bool IsolatedServer::ValidateTxn(const Transaction& tx, const Address& fromAddr,
   return true;
 }
 
+bool IsolatedServer::ValidateEthTxn(const Transaction& tx,
```

Comment:
Can we extract this method in a separate class in a separate file etc.?   I'd be great to eventually remove many of our modifications from IsolatedServer.cpp to a separate file, so that we exactly know the diff of EVM work and it is small in the preexisting code.

Reply:
perhaps in another PR though

Reply:
Sure, will do.